### PR TITLE
Add sqlite database

### LINKED ISSUE #1: Add sqlite as database to project
- [ ] Add sqlite as dependency to requirements file of api
- [ ] Connect local sqlite file to backend code
- [ ] Add possibility for user to choose whether a sqlite databse should be used at all (e.g. CLI parameter)

### PR DESCRIPTION
**Pull request by melissaSchmidt0302**
- [x] Add schema and sqlite database to api
- [x] Endpoint to create a database entry if entry cannot be found

Closes #1

### CODE REVIEW

File: api/app/db/database.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+DATABASE_URL = "sqlite:///./app/db/finance.db"
```
[the remark this answers is not in the document]
Comment (melissaSchmidt0302) [reply api/app/db/database.py:5]: for sure that might be a good idea. The next step would be to replace it by a centrally stored file like azure blob storage.

File: api/app/tickers/operations.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from sqlalchemy.orm import Session
+import json
+import yfinance
+
+from ..db import crud, schemas
+
+
+def save_finance_data(db: Session, stock: schemas.StockBase):
+    # Get data from yahoo
+    data = yfinance.Ticker(stock.symbol)
```
Comment (melissaSchmidt0302) [inline api/app/tickers/operations.py:11]: @leonox

File: api/app/main.py [hunk 3]
```diff
@@ -18,4 +18,9 @@
 
 @app.get('/')
 async def root():
-    return {'message': 'Hello from qupo'}
+    return {"message": "Hello from qupo"}
```
Comment (mathiashoeld) [inline api/app/main.py:21]: please be aware to use flake8-quotes for the correct use of quote signs

File: api/app/tickers/tickers.py [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
 
 
-@router.get('', tags=['tickers'])
+@router.get("/", tags=["tickers"])
 async def get_all_tickers():
-    return [{'name': 'ticker1'}, {'name': 'ticker2'}]
+    return [{"name": "ticker1"}, {"name": "ticker2"}]
+
+
+@router.post("/stock/", response_model=schemas.Stock)
```
Comment (mathiashoeld) [inline api/app/tickers/tickers.py:29]: this is a little confusing - the function name is `getStock`, but the endpoint is a `POST` endpoint

**Review by mathiashoeld — CHANGES_REQUESTED** [review]

**Pull request merged**